build.gradle.kts

```diff
@@ -166,7 +166,12 @@ gradle.taskGraph.whenReady {
 
 // == projects lists and flags ==
 // getters are required because of variable lazy initialization in Gradle
-val unpublishedProjects get() = setOf("benchmark", "guide", "kotlinx-serialization-json-tests")
+val unpublishedProjects get() = setOf(
+    "benchmark",
+    "guide",
+    "kotlinx-serialization-json-tests",
+    "test-protobuf",
```

Comment:
`proto-test-model` should be mentioned here

Reply:
Fixed